README.md

```diff
@@ -94,6 +97,183 @@ see [Developer’s guide to auth with Azure Resource Manager API](http://aka.ms/
 After creating the service principal, you should have three pieces of information, a client id (GUID), client secret
 (string) and tenant id (GUID) or domain name (string).
 
+# Azure Multiple API versions & Profiles
+
+With 0.15.0 of Azure SDK, multiple API versions and profiles are introduced. With these changes, each individual gem
+has multiple versions of the services and several profiles. The azure sdk rollup gem also consists of several profiles. The following section provides details on the usage of multiple API versions and profiles.
+
+## Why Multiple API versions?
+
+With the previous Azure SDK, you will have access to the latest (at the time of release) versions of our services. With 
+each new release, the versions may get updated. But, if you would like to continue using the old versions of some 
+services, such option was not available. But, with the multiple API versions, you could use any version of service. 
+
+## What is a  Profile?
+
+A profile is a combination of different resource types with different versions from different services. Using a profile,
+will help you mix and match between various resource types.
+
+## What to use?
+* If you would like to use the latest versions of **all** the services, then the recommendation is to use the **latest** profile of the Azure SDK rollup gem.
+
+* If you would like to use the services compatible with the **azure stack**, then the recommendation is to use the **v2017_03_09** profile of the Azure SDK rollup gem.
+
+* If you would like to use the latest version of **some** services, then the recommendation is to use the **latest** profile of those specific gems.
```

Comment:
should we say "latest api-version of _a_ service, then.." ?
I'd probably use the name of the profile "`Latest`" and say of the "specific gem" . I'd add "For example: <link to an example>"

Reply:
Done